lib/proto.js

```diff
@@ -108,8 +108,22 @@ exports.initialize = function(){
  */
 
 exports.enabled = function(facade){
-  return !!~ this.channels.indexOf(facade.channel())
-    && facade.enabled(this.name);
+  if (!facade.enabled(this.name)) {
+    return false;
+  }
+  // Pass through event for unbundled analytics.js integration if
+  // - this event came from the browser
+  // - this integration wasn't bundled in analytics.js
+  if (facade.channel() === 'client') {
+    // If the bundled metadata isn't included in the message, the client sending
+    // this message didn't support unbundling, so it must have already been sent
+    // from the client integration.
+    var bundled = facade.proxy('_metadata.bundled');
+    if (bundled && !bundled[this.name]) {
+      return true;
+    }
+  }
+  return this.channels.indexOf(facade.channel()) !== -1;
```

Comment:
Slightly more expressive:

https://github.com/ndhoule/includes

It's a native method in later node versions but not in the version integrations are using